crates/kubelet/src/lib.rs

```diff
@@ -55,6 +55,7 @@ pub(crate) mod kubeconfig;
 pub(crate) mod webserver;
 
 pub mod config;
+pub mod config_interpreter;
```

Comment:
Just to clarify: This is `pub` because of the trait implementation in `config_interpreter`?

Reply:
I thought it needed to be but it looks like having it private still works!  Fixed!  Thanks!